paasta_tools/autoscaling/autoscaling_service_lib.py

```diff
@@ -323,9 +324,26 @@ def get_error_from_utilization(utilization, setpoint, current_instances):
 
 def autoscale_marathon_instance(marathon_service_config, marathon_tasks, mesos_tasks):
     current_instances = marathon_service_config.get_instances()
-    if len(marathon_tasks) != current_instances:
-        write_to_log(config=marathon_service_config,
-                     line='Delaying scaling as marathon is either waiting for resources or is delayed')
+    too_many_instances_running = len(marathon_tasks) > int((1 + MAX_TASK_DELTA) * current_instances)
+    too_few_instances_running = len(marathon_tasks) < int((1 - MAX_TASK_DELTA) * current_instances)
+    if too_many_instances_running or too_few_instances_running:
```

Comment:
doesn't this mean we'll only scale up to the ``min_instances`` if the delta between ``min_instances`` and current_instances is > 30%?